frontend/src/app/pages/exercises/exercise/shared/trainer-toolbar/trainer-toolbar.component.html

```diff
@@ -9,8 +9,11 @@
     <button (click)="openTransferOverview()" class="btn btn-primary ms-3">
         Transferübersicht
     </button>
+    <button (click)="openAlarmGroupOverview()" class="btn btn-primary ms-3">
+        Alarmierungsgruppen
+    </button>
     <button
-        *ngIf="(exerciseStatus$ | async) === 'running'"
+        *ngIf="(exerciseStatus$ | async) !== 'running'"
```

Comment:
What?

Reply:
That seems to have happened while merging. Fixed now